src/doc/src/reference/specifying-dependencies.md

```diff
@@ -454,8 +454,51 @@ following to the above manifest:
 log-debug = ['bar/log-debug'] # using 'foo/log-debug' would be an error!
 ```
 
+### Inheriting a dependency from a workspace
+
+Dependencies can be inherited from a workspace by specifying the
+dependency in the workspace's [`[workspace.dependencies]`][workspace.dependencies] table.
+After that add it to the `[dependencies]` table with `dep.workspace = true`.
+
+The `workspace` key can be defined with:
```

Comment:
This wasn't entirely clear to me what it was meaning. Perhaps the following might be clearer?

```suggestion
Along with the `workspace` key, dependencies can also include these keys:
```